taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/ServerConceptSuggestionResource.java

```diff
@@ -0,0 +1,16 @@
+package com.digirati.taxman.rest.server;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import java.util.List;
+
+@ApplicationScoped
+@Path("/v0.1/concept-scheme/suggestions")
+public class ServerConceptSuggestionResource {
+
+    @GET
+    public List<String> getSuggestions() {
+        return List.of();
```

Comment:
I thought we weren't adding this until a future phase of development, but if it's needed from day 1 it should throw an exception so we don't miss it. If it's not needed until some indeterminate time in the future we should probably drop it from both this change and the API spec

Reply:
That's right. Removing this resource for now.